apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ReliableStreamReader.cs

```diff
@@ -77,7 +77,7 @@ public void Close()
                 return;
             }
             IsClosed = true;
-            _resultStream.Dispose();
+            (_resultStream as IDisposable)?.Dispose();
```

Comment:
Also handle `IAsyncDisposable`?

Reply:
Removed entirely. IAsyncStreamReader used to inherit from IDisposable, but even then the implementation was a no-op. I think it's reasonable not to dispose of this.